Tasks/PyPIPublisherV0/publisher.ts

```diff
@@ -20,7 +20,7 @@ var password = tl.getEndpointAuthorizationParameter(serviceEndpointId, 'password
 var text = util.format("[distutils] \nindex-servers =\n    pypi \n[pypi] \nrepository=%s \nusername=%s \npassword=%s", pythonServer, username, password);
 tl.writeFile(pypircFilePath, text, 'utf8');
 
-async function run(){
+(async () => {
```

Comment:
I had to adjust this task due to a problem in the @types file where a function had the same name.